# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1313602536 -7200
#      Wed Aug 17 19:35:36 2011 +0200
# Node ID be749686366052fe0b96ec0a1e7aa6a369bdb782
# Parent  83bcf326e9849184c8e3768510e217fd6afee9bf
Now using the document of the irc client as the model part of the conversation.

diff --git a/gui/src/IRCWidget.cpp b/gui/src/IRCWidget.cpp
--- a/gui/src/IRCWidget.cpp
+++ b/gui/src/IRCWidget.cpp
@@ -209,16 +209,17 @@ QWidget (parent)
            this, SLOT (handleUserNicknameChanged (QString)));
 
   connect (m_nickButton, SIGNAL (clicked ()), this, SLOT (showChangeUserNickPopup ()));
   connect (m_chatMessageTextEdit, SIGNAL (sendMessage (QString)),
            this, SLOT (sendMessage (QString)));
 
   m_chatMessageTextEdit->setCompleter
       (new QCompleter (m_ircClientInterface->ircChannelProxy ("#octave")->userListModel (), this));
+  m_chatWindow->setDocument (m_octaveChannel->conversationModel ());
 
   if (connectOnStartup)
     connectToServer ();
 }
 
 void
 IRCWidget::connectToServer ()
 {
